packages/terra-date-time-picker/src/terra-dev-site/doc/guides/DateTimePickerEvents.jsx

```diff
@@ -57,7 +57,7 @@ class DateTimePickerEvents extends React.Component {
         <p>
           onFocus Trigger Count:
           {' '}
-          <span id="blur-state">{this.state.focusTriggerCount}</span>
+          <span id="focus-state">{this.state.focusTriggerCount}</span>
```

Comment:
nit: Just to keep it consistent with the other IDs, you could changed the id to **"onFocus-count"** and also for blur count to **"onBlur-count"**

Reply:
Resolved in cc2c4e7